### PR TITLE
test: increase code coverage for article-summary

### PR DESCRIPTION
**Pull request by colincclark**
- added test coverage around article-summary

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/73b98f7831f44fae4fb0860f1e81758d19be870e/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/e99cb2144349cddcc118681471094367643c146b/index.html

File: packages/article-summary/__tests__/web/__snapshots__/article-summary.web.test.js.snap [hunk 1]
```diff
@@ -981,3 +1054,26 @@ exports[`Article Summary test on Web renders an article-summary with opinion byl
   </div>
 </div>
 `;
+
+exports[`Article Summary tests on Web should render ast correctly 1`] = `
```
Comment (craigbilner) [inline packages/article-summary/__tests__/web/__snapshots__/article-summary.web.test.js.snap]: looks a bit odd?

File: packages/article-summary/src/article-summary-renderer.js [hunk 2]
```diff
@@ -2,8 +2,8 @@ import React from "react";
 import { Text } from "react-native";
 
 export default {
-  paragraph(key, attributes, renderedChildren, indx) {
-    const padding = renderedChildren.length && indx !== 0 ? " " : "";
+  paragraph(key, attributes, renderedChildren, index) {
+    const padding = renderedChildren.length && index !== 0 ? " " : "";
```
Comment (colincclark) [inline packages/article-summary/src/article-summary-renderer.js]: @craigbilner how do I test the ternary? I am only getting the `true` result....

**Pull request merged**